src/classes/ADDR_Validator.cls

```diff
@@ -47,11 +47,8 @@ public with sharing class ADDR_Validator {
         if (addrId != null) {
             set<id> setAddrId = new Set<Id>{addrId};
             verifyRecords(setAddrId);
-            
-            PageReference pageRef = new PageReference('/' + addrId);
-            pageRef.setRedirect(true);
-            return pageRef;
-        } else return null;
+        }
```

Comment:
this page is really just for a button, I think, and it used to redirect back to the address object.  by now just returning null, I think it might land on the empty visualforce page.